.github/workflows/ci.yml

```diff
@@ -291,6 +291,34 @@ jobs:
           path: ${{ env.RUST_BIN_DIR }}/${{ matrix.binary }}
           retention-days: 1
 
+  cargo-vet:
+    name: Vet Dependencies
+    runs-on: ubuntu-latest
+    env:
+      CARGO_VET_VERSION: 0.3.0
+
+    steps:
+      - name: Checkout Source
+        uses: actions/checkout@v3
+
+      - name: Install Rust
+        run: rustup update stable && rustup default stable
```

Comment:
Can you use `cargo` action?

Reply:
Of course!

Reply:
Addressed on: https://github.com/infinyon/fluvio/pull/2795/commits/22f4edc921a3cf282dbb659be55b8bb5a5499111